python/tests/test_network.py

```diff
@@ -331,6 +331,16 @@ def test_buses(self):
                 bus.v_ang = 0.123456
                 self.assertEqual(bus.v_ang,0.123456)
 
+                # v max/min norm/emer violation limits set get
+                bus.v_max_norm = 1.234567
+                self.assertEqual(bus.v_max_norm,1.234567)
+                bus.v_min_norm = 0.901234
+                self.assertEqual(bus.v_min_norm,0.901234)
+                bus.v_max_emer = 1.234567
+                self.assertEqual(bus.v_max_emer,1.234567)
+                bus.v_min_emer = 0.901234
+                self.assertEqual(bus.v_min_emer,0.901234)
```

Comment:
also added checks for the aliases ``v_max`` and ``v_min`` set and gets and agains ``v_max_norm`` and ``v_min_norm``.